frontend/apps/reader/modules/readerrotation.lua

```diff
@@ -20,12 +20,12 @@ function ReaderRotation:registerKeyEvents()
         self.key_events = {
             -- these will all generate the same event, just with different arguments
             RotateLeft = {
-                { "J" },
+                { "Shift", "J" },
                 event = "Rotate",
                 args = -90
             },
             RotateRight = {
-                { "K" },
+                { "Shift", "K" },
```

Comment:
Not sure what these were supposed to do. They trigger on PDF, but just messing everything (blank page with current highlights rects).

Reply:
I don't think they do anything, not sure why they are there. Allegedly they rotate the screen or something like that, only they don't (thankfully)